example/main.go

```diff
@@ -5,11 +5,15 @@ import (
 	"errors"
 	"fmt"
 	"os"
-
-	"github.com/1password/onepassword-sdk-go"
 )
 
+// [developer-docs.sdk.go.sdk-import]-start
```

Comment:
Do we not want to include all the imports as aren't these required for our SDK? i.e. os - to get env variable, etc

Reply:
No, I don't think so, for two reasons:
* they currently aren't included, and I think a good first step for this migration would be to achieve parity before improving
* we shouldn't assume the user's setup - e.g. the SA token can be passed in multiple ways, not only envvars. a `context.Context` can be obtained from multiple libraries other than context, such as `xcontext`  - I think we should only include the non-variables here.

Reply:
Sounds good.